src/Sarif/Core/FileData.cs

```diff
@@ -17,7 +17,7 @@ public partial class FileData : ISarifNode
     {
         public static FileData Create(
             Uri uri, 
-            SarifWriters.LoggingOptions loggingOptions, 
+            OptionallyEmittedData dataToInsert = OptionallyEmittedData.None, 
```

Comment:
>OptionallyEmittedData [](start = 12, length = 21)

This change satisfies a long-standing concern I've had where we pass around a 'logging options' object for OM operations.